README.md

```diff
@@ -300,6 +300,36 @@ const MyAwesomeForm = () => {
 };
 ```
 
+## Validation helper
+
+In case you want to use several validation functions, there is `combineValidators` helper which can help you to set several validation functions for a field. This helper works with sync and async functions (by using promise for async).
```

Comment:
Suggestion:

> As it's a very common case to use several validation functions per field, we expose a `combineValidators` helper function that allow you to chain sync and async validation functions (it will run all of them sequentially until an error is returned).

Reply:
yes it sounds better to me 👍